src/Annotations/AppKernel.php

```diff
@@ -103,6 +102,15 @@ public function __construct(string $environment = 'dev')
                 'authority' => '',
                 'group' => '',
             ],
+            'common_mark.environment' => ($config['common_mark']['environment'] ?? []) + [
+                'allow_unsafe_links' => false,
+            ],
+            'html_purifier' => ($config['html_purifier'] ?? []) + [
+                'Cache.SerializerPath' => __DIR__.'/../../var/cache/html_purifier',
```

Comment:
Could we extract the cache path as its own parameter?

Reply:
You mean `var/cache/` or `/srv/annotations/var/cache`?

Reply:
Something like https://github.com/elifesciences/annotations/blob/83c91e5ea7e296cd52c1df0fec4c862431268388/src/Annotations/AppKernel.php#L81